source/class.cpp

```diff
@@ -1080,6 +1080,11 @@ string ClassBinder::bind_nested_classes(Context &context)
 {
 	string c;
 	for_public_nested_classes([&c, &context, this](CXXRecordDecl const *innerC) {
+		if ( !innerC->isFirstDecl() ) return;
```

Comment:
Could you please add comment for this line stating that it is need to skip forward declarations to avoid repeated bindings? Thanks,

Reply:
Done!